### ISSUE TITLE
anzu--query-replace-common: Wrong type argument when repeating previous replace. 

### ISSUE THREAD

**Issue by pronobis** [syohex/emacs-anzu#49]
Whenever I use anzu-query-replace to repeat a previous query-replace event without providing new values, I get the error:
```
anzu--query-replace-common: Wrong type argument: stringp, ("a" . "b")
```
This seems to occur in the line 698 in:
```
(string-match "\0" from)
```
since from is not a string but ("a" . "b").

**Comment by syohex** [comment]
Thanks for report.
I have fixed at #50. Please check latest version.

**Comment by pronobis** [comment]
Works perfect! Thanks!
